src/__tests__/__snapshots__/plugin.js.snap

```diff
@@ -0,0 +1,644 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`gatsby-remark-embedder can transform supported links 1`] = `
```

Comment:
I strongly advise against huge snapshots like this (https://kcd.im/snapshots).

If you want to snapshot something, then you could snapshot the generated HTML, not the AST.

Reply:
I snapshotted the AST, since that's what the plugin is actually handling.

Will read the blogpost and try again with smaller fixtures 🙂

Reply:
I'd recommend printing the AST to HTML and snapshotting that. That will be much easier to maintain (whereas any diff change in the AST would be impossible to review effectively).

Reply:
To HTML or just stringify (which will create the Markdown version) it again? 🤔

Reply:
You'd need to pass the markdown AST to the remark HTML plugin, then snapshot the result of that.

Reply:
`remark.stringify(markdownAST)` is what I meant, which will give you a Markdown string

```md
https://twitter.com/user/status/123
```
-> `remark.parse`
```json
{
  // AST comes here
}
```
-> `gatsby-remark-embedder`
```json
{
  // transformed AST comes here
}
```
-> `remark.stringify`
```md
<iframe src="https://twitter.com/user/status/123"></iframe>
```

Reply:
And what I'm saying is that it's *very* hard to maintain snapshots of ASTs. It's basically impossible to review changes to the snapshot. Nobody bothers, so the tests are worse than no tests at all because they make you think you don't need helpful tests because you already "have tests."

I'd rather have no tests, than tests with snapshots like this.

Reply:
Oh oh oh, sorry, I was confused by your comment. Yes, I think `remark.stringify` is good 👍

Reply:
No problem, probably my bad explanation 😉

Reply:
No problem, probably my bad explanation 😉

Reply:
@kentcdodds I've updated the tests 🙂